electron/yarn.lock

```diff
@@ -97,10 +97,10 @@
   dependencies:
     jsrsasign "8.0.12"
 
-"@wireapp/commons@1.4.0":
-  version "1.4.0"
-  resolved "https://registry.yarnpkg.com/@wireapp/commons/-/commons-1.4.0.tgz#2d3fff24bad6b89c463a3bd6dba11d3b6a3669bc"
-  integrity sha512-rmhEDevsCgVmKzB5d9cyhwKu/+HmULwEPPU8aRK3Uf2CUIXZJMjI1V8BEOpQBzwF9pMitiq8Wiqt8+pvXXoGjA==
+"@wireapp/commons@1.4.1":
```

Comment:
Is this part of the PR?

Reply:
Yes because it was not correctly updated in the previous PR (https://github.com/wireapp/wire-desktop/pull/2375) so yarn fixed it when running `yarn install`.